airbyte-integrations/connectors/destination-s3/src/test-integration/java/io/airbyte/integrations/destination/s3/S3CsvDestinationAcceptanceTest.java

```diff
@@ -88,21 +65,8 @@
     return fieldTypes;
   }
 
-  private static Optional<Number> parseNumber(String input) {
-    try {
-      double number = Double.parseDouble(input);
-      if (number % 1 == 0) {
-        return Optional.of((int) number);
-      } else {
-        return Optional.of(number);
-      }
-    } catch (Exception e) {
-      return Optional.empty();
-    }
-  }
```

Comment:
This is the arbitrary type conversion I was talking about in the comment about changing integer HKD field to decimals. When the testing data has consistent typing, such conversion can be removed.